R/dbetabinom.R

```diff
@@ -157,7 +157,7 @@ dbetaMix <- Vectorize(dbetaMix, vectorize.args = "x")
 #' @example examples/pbetaMix.R
 #' @export
 pbetaMix <- function(q, par, weights, lower.tail = TRUE) {
-  assert_number(q, lower = 0, upper = 1, finite = TRUE)
+  assert_numeric(q, lower = 0, upper = 1, finite = TRUE)
```

Comment:
It would be better to keep this as `assert_number`, or is it breaking anything? I think if you had a true mixture of beta distributions you could pass in a `q` like this and it would be incorrect.